pm/recipient.go

```diff
@@ -269,7 +278,13 @@ func (r *recipient) faceValue(sender ethcommon.Address) (*big.Int, error) {
 		// Set faceValue = maxFloat
 		faceValue = maxFloat
 	}
-
+	
+	if r.maxfacevalue.Cmp(big.NewInt(0)) > 0 {
+		//if faceValue > maxfacevalue, set faceValue=maxfacevalue
+		if r.maxfacevalue.Cmp(faceValue) < 0 {
```

Comment:
```suggestion
		if r.maxfacevalue.Cmp(faceValue) < 0 {
```

Nit: There is no need for this comment.

Reply:
Agree, not used to Go with using .Cmp.  Will remove comment.